tgui/packages/tgui/interfaces/Pod/constants.ts

```diff
@@ -0,0 +1,19 @@
+// import Example from './Parts/Example';
+import CargoHold from './Parts/CargoHold';
+import DNAPart from './Parts/DNAPart';
+import GenericLines from './Parts/GenericLines';
+import Gun from './Parts/Gun';
+import OreHold from './Parts/OreHold';
+import PINPart from './Parts/PINPart';
+import WarpDrive from './Parts/WarpDrive';
```

Comment:
these should really be a named import, `import { CargoHold }` etc. export default is good for page routers (which we don't use) only. please just export the modules without `default` and import them like this

https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910

Reply:
i dont know how to apply that article sorry